src/webportal/src/app/user/user-view/user-view.component.js

```diff
@@ -250,6 +258,174 @@ const updateUserGithubPAT = (username) => {
   });
 };
 
+const downloadTemplate = () => {
+  let csvString = csvParser.unparse([{
+    [columnUsername]: 'student1',
+    [columnPassword]: '111111',
+    [columnAdmin]: false,
+    [columnVC]: 'default',
+    [columnGithubPAT]: '',
+  }]);
+  let universalBOM = '\uFEFF';
```

Comment:
Is BOM required in all systems in CSV standard or just for Windows?

Reply:
BOM is required for Excel (I assume the user on Windows use Excel to edit the CSV file).
And on the other hand, vim on Linux and VS code on both Linux and Windows can well deal the BOM header.